core/meta/inc/TInterpreter.h

```diff
@@ -162,6 +162,7 @@ class TInterpreter : public TNamed {
                                void (*histaddFunc)(const char* line)) = 0;
    virtual void     Reset() = 0;
    virtual void     ResetAll() = 0;
+   virtual void     SetupCoreModules() = 0;
```

Comment:
`TCling` doesn't seem to implement this? Is this interface actually needed, or can this (assuming "this" is really `LoadCoreModules()`) be a private function of `TCling` or even a static function in TCling.cxx?

Reply:
Valid point, that was an artifact of the first version of this patch. Thanks! It's now a static function in TCling.